js/shared/i18n.js

```diff
@@ -82,7 +82,7 @@ import Jed from "jed";
         // Replace the substitutions with the appropriate option
         for (var i = 1; i < split.length; i += 2) {
             var replaceWith = options[split[i]];
-            split[i] = _.isUndefined(replaceWith) ?
+            split[i] = (replaceWith === undefined) ?
```

Comment:
This is what i18n has in webapp, seemed better to change it than bring in underscore.

Reply:
sgtm... we should really create our own i18n npm package so that it's easy to use here and elsewhere.

Reply:
it looks like this file was just assuming that `_` was a global.  I'm glad we're moving past those dark days.